crates/sui-tool/src/commands.rs

```diff
@@ -53,30 +56,24 @@ pub enum ToolCommand {
         #[clap(long = "history", help = "show full history of object")]
         history: bool,
 
-        /// Concise mode prints tabular output suitable for processing with unix tools. For
+        /// Concise mode groups responses by results.
+        /// prints tabular output suitable for processing with unix tools. For
         /// instance, to quickly check that all validators agree on the history of an object:
         ///
         ///     $ sui-tool fetch-object --id 0x260efde76ebccf57f4c5e951157f5c361cde822c \
         ///         --genesis $HOME/.sui/sui_config/genesis.blob \
-        ///         --history --concise   --no-header \
-        ///         | sort -k2 -k1 \
-        ///         | uniq -f1 -c
+        ///         --history --concise
```

Comment:
I actually use that `sort | uniq` incantation and I would like to keep being able to use it - the entire reason I added the `--concise` mode was to be able to process the output with unix tools. this change breaks that. I definitely agree that the new output is prettier when viewed as a whole, however.

Can you simply add a third output mode? Perhaps make your new output the default, and make the current default output triggered with `--verbose`?

Reply:
PR updated with `verbosity` param replacing `concise`. It supports 3 modes "groupped" (default), "concise" and "verbose"